mongodb-on-ubuntu/azuredeploy.json

```diff
@@ -42,17 +36,47 @@
     },
     "imageSKU": {
       "type": "string",
-      "defaultValue": "14.04.5-LTS",
+      "defaultValue": "18.04-LTS",
       "metadata": {
         "description": "Image SKU"
       }
     },
     "vmSize": {
       "type": "string",
-      "defaultValue": "Standard_A0",
+      "defaultValue": "Standard_A1_v2",
       "metadata": {
         "description": "VM Size"
-      }
+      },
+      "allowedValues": [
```

Comment:
remove this since it creates a maint problem later

Reply:
you mean- allowedValues? or new default value?

Reply:
remove the allowedValues property

Reply:
@bmoore-msft done